Changelog.md

```diff
@@ -49,6 +49,7 @@ Breaking Changes:
  * Type Checker: Fallback function must be external. This was already the case in the experimental 0.5.0 mode.
  * Type Checker: Interface functions must be declared external. This was already the case in the experimental 0.5.0 mode.
  * Type Checker: Address members are not included in contract types anymore. An explicit conversion is now required before invoking an ``address`` member from a contract.
+ * Type Checker: Inline-assembly's loose error type raised from ``Warning`` to ``SyntaxError``.
```

Comment:
I think this is not clear for users to understand. Something like `Disallow "loose assembly` syntax entirely. This means that jumps and jump labels cannot be used anymore.`